habitat/tasks/nav/nav_task.py

```diff
@@ -363,6 +443,148 @@ def update_metric(self, episode, action):
             self._metric += 1
 
 
+class TopDownMap(habitat.Measure):
+    """Top Down Map measure
+    """
+
+    def __init__(self, sim: Simulator, config: Config):
+        self._sim = sim
+        self._draw_source_and_target = config.DRAW_SOURCE_AND_TARGET
+        self._draw_border = config.DRAW_BORDER
+        self._grid_delta = 3
+        self._step_count = 0
+        self._max_steps = config.MAX_EPISODE_STEPS
+        self._map_resolution = (config.MAP_RESOLUTION, config.MAP_RESOLUTION)
+        self._num_samples = 20000
```

Comment:
Move this inside top down map config.

Reply:
What do you mean by this? Like move it into the defaults.py config somewhere?

Reply:
Fixed? I'm still not really understanding what you guys want.

Reply:
We want all consts to be part of config:
`self._num_samples = 20000 ` => `_C.TASK.TOP_DOWN_MAP.NUM_SAMPLES = 20000`.
Same for `_grid_delta`.

Reply:
I see. Done